WBooks/Books_Table/BooksTableViewController.swift

```diff
@@ -94,4 +94,13 @@ extension BooksTableViewController: UITableViewDelegate, UITableViewDataSource {
         
         return cell
     }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        _view.tableView.deselectRow(at: indexPath, animated: true)
+        
+        let book = _booksViewModel.books.value[indexPath.row]
+        
+        let detailsView = BookDetailsViewController(bookViewModel: _booksViewModel.createBookDetailsViewModel(book: book))
+        navigationController?.pushViewController(detailsView, animated: true)
```

Comment:
I would use `!` because we expect there to be a value to use, if not it won't work as expected and we wouldn't know why.